ceph.spec.in

```diff
@@ -81,24 +81,20 @@ BuildRequires:	python
 BuildRequires:	python-argparse
 BuildRequires:	python-nose
 BuildRequires:	python-requests
-%if ( 0%{?rhel} > 0 && 0%{?rhel} < 7 ) || ( 0%{?centos} > 0 && 0%{?centos} < 7 )
-BuildRequires:	python-sphinx10
-%endif
```

Comment:
Yes, @ktdreyer told me to make that change in https://github.com/ceph/ceph/pull/4881#discussion-diff-31933581 -- github is hiding this under the cryptic message "ktdreyer commented on an outdated diff"